test/e2e/e2e_test.go

```diff
@@ -207,10 +220,82 @@ func requireReplicatedStream(t *testing.T, c1 *catalystContainer, c2 *catalystCo
 		content := string(body)
 		for _, expected := range []string{"RESOLUTION=1920x1080", "FRAME-RATE=30", "index.m3u8"} {
 			if !strings.Contains(content, expected) {
+				glog.Info("Failed to get HLS manifest")
 				return false
 			}
 		}
+		glog.Info("Got HLS manifest!")
 		return true
 	}
 	require.Eventually(t, correctStream, 5*time.Minute, time.Second)
+
+	// Define a set of protocols to test at output of node catalyst-two
+	protocols := map[string]string{
+		"hls": fmt.Sprintf("http://localhost:%s/hls/stream+foo/index.m3u8", c2.http),
+		// add more protocol <-> url mapping here to test in the future
+	}
+
+	// rm any old output files from previous MistLoadTest runs
+	cleanupFiles("*json")
+	cleanupFiles("*html")
+
+	// For each protocol defined above, check for replicated stream output
+	// at node catalyst-two (implicitly tests DTSC path in-between the nodes
+	for prot, url := range protocols {
+
+		glog.Infof("Testing %s using url: %s", prot, url)
+
+		cmdMistLoadTest := exec.Command("../../../bin/./MistLoadTest", url)
+		out, err := cmdMistLoadTest.Output()
+		require.NoError(t, err)
+		if err != nil {
+			glog.Fatalf("Failed to start MistLoadTest: %s", err)
+		}
+		glog.Infof("MistLoadTest stdout: %s", out)
+
+		defer cmdMistLoadTest.Process.Kill()
+
+	}
+
+	// Parse output results for each protocol tested above
+	var p bool = true
+	for prot := range protocols {
+
+		matches, err := filepath.Glob("*" + strings.ToUpper(prot) + "*json")
+		if err != nil {
+			panic(fmt.Errorf("Glob failed: %w", err))
```

Comment:
Will defer to @leszko here on best practices for tests like this, but should this maybe be returning an error instead of panicking? Seems like a good practice in general... on the other hand I can see how this error would be like "test harness failure" instead of "failing test" and maybe a panic makes sense there.

Reply:
Yeah, I'm inclined to remove the panic and just make it a general "fail". We'd be reviewing logs anyways to see where the failures are - doesn't seem like a panic would be anymore useful here.